tests/integration/test_fim/test_files/test_follow_symbolic_link/data/test_cases/cases_disabled.yaml

```diff
@@ -2,88 +2,118 @@
   description: When a real-time monitored directory es deleted
                or added FIM inmediately raises a log.
   configuration_parameters:
-    TEST_DIR: !!python/object/apply:os.path.join [/test_dir, symlink]
+    TEST_DIR: !!python/object/apply:os.path.join
+          args: [!!python/object/apply:os.getcwd [],test_dir, symlink]
     FREQUENCY: 43200 # As default 12 hs
     FIM_MODE:
       realtime: 'yes'
   metadata:
-    folder_to_monitor: !!python/object/apply:os.path.join [/test_dir]
-    symlink:  !!python/object/apply:os.path.join [/test_dir, symlink]
-    symlink_target:  !!python/object/apply:os.path.join [/test_dir]
-    symlink_new_target:  !!python/object/apply:os.path.join [/test_dir, new_target]
+    folder_to_monitor: !!python/object/apply:os.path.join
+              args: [!!python/object/apply:os.getcwd [], test_dir]
+    symlink:  !!python/object/apply:os.path.join
+              args: [!!python/object/apply:os.getcwd [], test_dir, symlink]
+    symlink_target:  !!python/object/apply:os.path.join
+              args: [!!python/object/apply:os.getcwd [], test_dir]
+    symlink_new_target: !!python/object/apply:os.path.join
+              args: [!!python/object/apply:os.getcwd [], test_dir, new_target]
     fim_mode: realtime
 
 - name: Symlink follow disabled - Real-time
   description: When a real-time monitored directory es deleted
                or added FIM inmediately raises a log.
   configuration_parameters:
-    TEST_DIR: !!python/object/apply:os.path.join [/test_dir, symlink]
+    TEST_DIR: !!python/object/apply:os.path.join
+          args: [!!python/object/apply:os.getcwd [],test_dir, symlink]
```

Comment:
add a blank space after the first comma "[], test_dir".

Reply:
Done!